regression-test/suites/schema_change_p0/test_add_keys_light_schema_change.groovy

```diff
@@ -0,0 +1,454 @@
+                break
+            } else {
+                sleep(100)
+                if (max_try_time < 1){
+                    assertEquals(1,2)
+                }
+            }
+        }
+        return jobStateResult
+    }
+    
+    def tableName = "add_keys_light_schema_change"
+    try {
+
+        String backend_id;
+        def backendId_to_backendIP = [:]
+        def backendId_to_backendHttpPort = [:]
+        getBackendIpHttpPort(backendId_to_backendIP, backendId_to_backendHttpPort);
+
+        backend_id = backendId_to_backendIP.keySet()[0]
+        def (code, out, err) = show_be_config(backendId_to_backendIP.get(backend_id), backendId_to_backendHttpPort.get(backend_id))
+        
+        logger.info("Show config: code=" + code + ", out=" + out + ", err=" + err)
+        assertEquals(code, 0)
+        def configList = parseJson(out.trim())
+        assert configList instanceof List
+
+        boolean disableAutoCompaction = true
+        for (Object ele in (List) configList) {
+            assert ele instanceof List<String>
+            if (((List<String>) ele)[0] == "disable_auto_compaction") {
+                disableAutoCompaction = Boolean.parseBoolean(((List<String>) ele)[2])
+            }
+        }
+        int max_try_time = 3000
+        List<List<Object>> table_tablets = null;
+        List<List<Object>> rowset_cnt = null;
+
+        // case 1.1: light schema change type check: add key column in short key column num
+        sql """ DROP TABLE IF EXISTS add_keys_light_schema_change """
+        sql """
+                CREATE TABLE IF NOT EXISTS add_keys_light_schema_change (
+                    `user_id` LARGEINT NOT NULL COMMENT "用户id",
+                    `date` DATEV2 NOT NULL COMMENT "数据灌入日期时间",
+                    `city` VARCHAR(20) COMMENT "用户所在城市",
+                    `age` SMALLINT COMMENT "用户年龄",
+                    `sex` TINYINT COMMENT "用户性别",
+
+                    `cost` BIGINT SUM DEFAULT "0" COMMENT "用户总消费",
+                    `max_dwell_time` INT MAX DEFAULT "0" COMMENT "用户最大停留时间",
+                    `hll_col` HLL HLL_UNION NOT NULL COMMENT "HLL列",
+                    `bitmap_col` Bitmap BITMAP_UNION NOT NULL COMMENT "bitmap列")
+                AGGREGATE KEY(`user_id`, `date`, `city`, `age`, `sex`) DISTRIBUTED BY HASH(`user_id`)
+                BUCKETS 4
+                PROPERTIES ( "replication_num" = "1", "light_schema_change" = "true");
+            """
+        sql """ 
+                INSERT INTO add_keys_light_schema_change VALUES
+                (1, '2017-10-01', 'Beijing', 10, 1, 1, 20, hll_hash(1), to_bitmap(1))
+            """
+        sql """
+            ALTER TABLE add_keys_light_schema_change ADD COLUMN new_key_column INT default "2" AFTER user_id PROPERTIES ("timeout"="604800")
+            """
+        
+        def jobStateResult = waitJobFinish(""" SHOW ALTER TABLE COLUMN WHERE IndexName='add_keys_light_schema_change' ORDER BY createtime DESC LIMIT 1 """, 9)
```

Comment:
Do not need wait, because light weight schema change is sync process. Not need background process.

Reply:
jobStateResult is used to check schema change is use light schema change or not.